### ISSUE TITLE
Move sql file actions to vscode tab bar

### ISSUE THREAD

**Issue by mojtabakaviani** [dbcodeio/public#4]
At current sql file actions with icons is in top of file and very small, if move to right of vscode tab bar is very better!

**Comment by mikeburgh** [comment]
Yes, the actions/icons at the top of the file are using codelens. You can increase the font size with the setting: editor.codeLensFontSize

There should also be a play icon on the right in the vscode tab bar which will show on any .sql file you have open, it will execute the document (or selection).

<img width="1101" alt="Screenshot 2024-04-09 at 10 12 04 AM" src="https://github.com/dbcodeio/public/assets/1918994/307dea68-54aa-43dc-bb21-fe6f00e5f99f">


You can also use the keyboard shortcut (cmd/ctrl + D + E).

Ideally we would like to see vscode add a proper toolbar, but as yet they have not done that, see: https://github.com/microsoft/vscode/issues/41309
